releases/v15_0.rst

```diff
@@ -0,0 +1,28 @@
+#######################
+LSST Stack 15.0 Release
+#######################
+
+Version 15.0 of the LSST Science Pipelines is now available. It is based on 2018 weekly 10.
+
+LSST Pipelines
+==============
```

Comment:
What are we actually releasing? “LSST Stack”? “LSST Science Pipelines”? “LSST Pipelines”? They're all kinda the same, but the naïve reader running into this for the first time isn't going to understand that. Can we come up with a single term (I think “LSST Science Pipelines” is fine) and use it consistently in this text?

(I don't want to “name the codebase” — that's a whole other discussion — just to make this page less confusing.)

Reply:
:+1: "LSST Science Pipelines"